### PR TITLE
[1LP][RFR] first retirement tests from global region

### PR DESCRIPTION
**Pull request by izapolsk**
based on #9191

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 77bd375acac0b8a6a4584c30743bc5f0e9191626

The *global* fixture **`temp_appliances_unconfig_modscope_rhevm`** is used in the following files:

The *global* fixture **`multi_region_cluster`** is used in the following files:
- **cfme/tests/test_login.py**
    - *test_multiregion_displayed_on_login*

The *global* fixture **`setup_global_appliance`** is used in the following files:

The *global* fixture **`setup_remote_appliances`** is used in the following files:

The *global* fixture **`setup_multi_region_cluster`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_provisioning.py**
    - *test_provision_from_template_from_global_region*
- **cfme/tests/cloud_infra_common/test_retirement.py**
    - *test_vm_retirement_from_global_region*
- **cfme/tests/test_login.py**
    - *test_multiregion_displayed_on_login*

The *global* fixture **`setup_remote_provider`** is used in the following files:

The *global* fixture **`activate_global_appliance`** is used in the following files:

Please, consider creating a PRT run against these tests make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 98b15ed019342e4b78bbe103a2b41a479f8dca0a

The *global* fixture **`temp_appliances_unconfig_modscope_rhevm`** is used in the following files:

The *global* fixture **`multi_region_cluster`** is used in the following files:
- **cfme/tests/test_login.py**
    - *test_multiregion_displayed_on_login*

The *global* fixture **`setup_global_appliance`** is used in the following files:

The *global* fixture **`setup_remote_appliances`** is used in the following files:

The *global* fixture **`setup_multi_region_cluster`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_provisioning.py**
    - *test_provision_from_template_from_global_region*
- **cfme/tests/cloud_infra_common/test_retirement.py**
    - *test_vm_retirement_from_global_region*
- **cfme/tests/test_login.py**
    - *test_multiregion_displayed_on_login*

The *global* fixture **`setup_remote_provider`** is used in the following files:

The *global* fixture **`activate_global_appliance`** is used in the following files:

Please, consider creating a PRT run against these tests make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/cloud_infra_common/test_retirement.py [hunk 1]
```diff
@@ -298,12 +296,45 @@ def test_resume_retired_instance(retire_vm, provider, remove_date):
...
+        testSteps:
+            1. Have a VM created in the provider in the Remote region
+               subscribed to Global.
+            2. Retire the VM using the Global appliance.
+        expectedResults:
+            1.
+            2. VM transitions to Retired state in the Global and Remote region.
+
+    """
+    retire_times = dict()
```
Comment (john-dupuy) [inline cfme/tests/cloud_infra_common/test_retirement.py]: Why not just?
```python
retire_times = {"start": generate_retirement_date_now() + timedelta(minutes=-5)}
```

Comment (izapolsk) [reply cfme/tests/cloud_infra_common/test_retirement.py]: this is copypasta of retire_now test. It's just run against multi-region appliance

**Comment by dajoRH** [comment]
I detected some fixture changes in commit fe26d9a726a13729474f2a4f35acee55444fe890

The *global* fixture **`temp_appliances_unconfig_modscope_rhevm`** is used in the following files:

The *global* fixture **`multi_region_cluster`** is used in the following files:
- **cfme/tests/test_login.py**
    - *test_multiregion_displayed_on_login*

The *global* fixture **`setup_global_appliance`** is used in the following files:

The *global* fixture **`setup_remote_appliances`** is used in the following files:

The *global* fixture **`setup_multi_region_cluster`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_provisioning.py**
    - *test_provision_from_template_from_global_region*
- **cfme/tests/cloud_infra_common/test_retirement.py**
    - *test_vm_retirement_from_global_region*
- **cfme/tests/test_login.py**
    - *test_multiregion_displayed_on_login*

The *global* fixture **`setup_remote_provider`** is used in the following files:

The *global* fixture **`activate_global_appliance`** is used in the following files:

Please, consider creating a PRT run against these tests make sure your fixture changes do not break existing usage :smiley:

**Pull request merged**